webapp/sass/responsive/_mobile.scss

```diff
@@ -209,11 +209,12 @@
 
         .post__header {
             margin-bottom: 0;
+            padding-right: 50px;
```

Comment:
@kaifaust Then we can just reduce the popover width for lower resolutions by 5-10 pixels, so no matter how long the name is, at least for the responsive view, it remains in the same line? Also I'm looking specifically at the right hand side comment view, the center channel looks fine.